### ISSUE TITLE
Docuent Phone Requirement

### ISSUE THREAD

**Issue by maltfield** [eneshecan/whatsapp-for-linux#374]
This issue is an attempt to address the obvious question: does this `whatsapp-for-linux` serve as an *alternative* to the WhatsApp app (ie: "is it a standalone desktop app?") or is it just yet-another-second-class-desktop-client that routes messages through the phone on the intranet?

### Solution

Please do *not* answer this question on this GitHub issue. I'm pretty sure that this is a second-class app that requires a phone

 * https://github.com/eneshecan/whatsapp-for-linux/issues/30

The solution to this ticket should be to state, very clearly, in the `README.md` file if:

1. This `whtasapp-for-linux` still requires the user to first install and create an account in the official WhatsApp app, and all messages are routed through the phone or (eg like whatsapp web)
2. This `whtasapp-for-linux` still requires the user to first install and create an account in the official WhatsApp app, but afterwords WhatsApp can be uninstalled forever, and the desktop client can replace it (eg like Telegram)
3. This `whtasapp-for-linux` is an entirely first-class desktop app. You can create an account in it directly, and you never need to use the official WhatsApp mobile app for any reason.

Again, please add this information to the `README.md` file -- not to this ticket.

**Comment by AntonioFasano** [comment]
Your notion of second-class-app is highly subjective. For example, to me, the email apps are first-class apps, due to their flexibility, and the various proprietary messaging apps are second-class.

Be it as it may, Meta is a listed company, not a foundation, thus their app is tied to the phone, where they might have access to users contacts, locations, media, etc. from which they can monetize from.

That being the case, WhatsApp for Linux [has the abilities of WhatsApp Web client](https://github.com/eneshecan/whatsapp-for-linux/wiki), which will be logged out if you don't use your phone for over 14 days. See [https://faq.whatsapp.com/668538004658079?helpref=faq_content](faq.whatsapp.com/668538004658079?helpref=faq_content)

**Comment by maltfield** [comment]
If this repo's app uses WhatsApp Web and "will be logged out if you don't use your phone for over 14 days", then that should be documented.

To be clear: writing here in the issue isn't documentation. These limitations should be clearly added to the repo's `README.md` or other documentation.

**Comment by AntonioFasano** [comment]
I don't think so. WhatsApp Web is a Meta product. Users wanting to learn about it should look at their docs. 
Perhaps here, they might add a "based on WhatsApp Web" to enlighten less tech-savvy users.

**Comment by magdesign** [comment]
for those people searching for the answer if this **runs on linux mobile**: yes, but only usable in landscape mode since in portrait you wil see only the contacts:
![20240728_212422](https://github.com/user-attachments/assets/e8d166a1-44b7-47eb-9cb4-d1a33bd9e998)
![20240728_212408](https://github.com/user-attachments/assets/2f7007f6-c5c9-416e-ae60-715fa005817c)
